Imgsmall.java

```diff
@@ -0,0 +1,34 @@
+import javax.imageio.ImageIO;
+import java.awt.*;
+import java.awt.image.BufferedImage;
+import java.awt.image.RenderedImage;
+import java.io.File;
+import java.net.URL;
+
+public class Imgsmall {
+  public static void main(String[] args) throws Exception{
+    //get url object
+    URL url = new URL(args[0]);
+    float small = (100-Integer.parseInt(args[1]))/100;
+    //System.out.println(small);
+
+    //get image from url
+    BufferedImage img = ImageIO.read(url);
```

Comment:
Similarly see what Exception is thrown here - what if the URL is not an image (e.g. http://www.google.com - a valid URL but not an image).